### PR TITLE
add MapTest application

### PR DESCRIPTION
**Pull request by ruslan-shtukar-axon**

### CODE REVIEW

File: MapTest/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "com.android.support:appcompat-v7:$supportVersion"
+    implementation "com.android.support.constraint:constraint-layout:$supportConstraintVersion"
+    implementation "com.google.android.gms:play-services-maps:$gmsPlayServicesVersion"
+    implementation "com.google.android.gms:play-services-location:$gmsPlayServicesVersion"
+    implementation "com.google.android.gms:play-services-places:$gmsPlayServicesVersion"
+    implementation "com.google.maps:google-maps-services:0.2.4"
```
Comment (ghost) [inline MapTest/app/build.gradle]: Please, move the version to the version.gradle file.

File: MapTest/app/src/main/java/com/rshtukaraxondevgroup/maptest/view/MapsActivity.java [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        mapFragment.getMapAsync(this);
+
+        mButtonDirection.setOnClickListener(v -> onDirectionClick());
+        mBottomNavigationView.setOnNavigationItemSelectedListener(
+                item -> {
+                    switch (item.getItemId()) {
+                        case R.id.action_bank:
+                            if (mMarker != null) {
+                                mPlacesMapPresenter.downloadMapPlaces(mMarker.getPosition().latitude,
+                                        mMarker.getPosition().longitude, "bank");
```
Comment (ghost) [inline MapTest/app/src/main/java/com/rshtukaraxondevgroup/maptest/view/MapsActivity.java]: Move "bank, restaurant" etc. to the strings.xml file.

File: MapTest/app/src/main/java/com/rshtukaraxondevgroup/maptest/view/MapsActivity.java [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+            ActivityCompat.requestPermissions(this,
+                    new String[]{Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_FINE_LOCATION},
+                    MY_PERMISSION_ACCESS_COURSE_LOCATION);
+        } else if (mMap != null) {
+            mMap.setMyLocationEnabled(true);
+        }
+    }
+
+    private void showDefaultLocation() {
+        Toast.makeText(this, "Location permission not granted, " + "showing default location",
```
Comment (ghost) [inline MapTest/app/src/main/java/com/rshtukaraxondevgroup/maptest/view/MapsActivity.java]: Do not use concatenation. It is one string.

File: MapTest/app/build.gradle (hunk 1 above)
Comment (ruslan-shtukar-axon) [reply MapTest/app/build.gradle]: ok

File: MapTest/app/src/main/java/com/rshtukaraxondevgroup/maptest/view/MapsActivity.java (hunk 2 above)
Comment (ruslan-shtukar-axon) [reply MapTest/app/src/main/java/com/rshtukaraxondevgroup/maptest/view/MapsActivity.java]: ok

File: MapTest/app/src/main/java/com/rshtukaraxondevgroup/maptest/view/MapsActivity.java (hunk 3 above)
Comment (ruslan-shtukar-axon) [reply MapTest/app/src/main/java/com/rshtukaraxondevgroup/maptest/view/MapsActivity.java]: ok

**Pull request merged**